### PR TITLE
SRM-12650

### PR DESCRIPTION
**Pull request by SudheendraKatte**
Updated the tooltip for the following fields:

**Create or select an Environment**
Environment represents your deployment targets logically (QA, Prod, etc). You can select an existing environment or create a new one.

**Create or select a Service**
A service represents your micro service and other workloads logically. You can select an existing service or create a new one.

**Description (optional)**
Enter a description.

**Monitored service name**
This field is autopopulated by combining the Service and Environment names.

**Tags (optional)**
You can assign an existing tag or create a new tag. Tags are metadata and they can contain any character. You can use tags to search and filter the monitored service.

**Id**
Updated the link to point to Developer Hub.

### CODE REVIEW

**Comment by Thani80** [comment]
@SudheendraKatte Tooltip content looks good to me.

File: src/Tooltip/TooltipDictionary.yaml [hunk 1]
```diff
@@ -3591,15 +3591,13 @@ monitoredServiceType:
     content: Select the Service type. You can choose Application or Infrastructure.
     width: "150"
 serviceSelectOrCreate:
-    content: Select a user journey from the list. To create a new user journey, click Add New. A user journey is a visualization of a user's interaction with a service to achieve a specific goal over time and across different channels.
+    content: A service represents your micro service and other workloads logically. You can select an existing service or create a new one.    
     width: "300"
```
Comment (SmritiSatya) [inline src/Tooltip/TooltipDictionary.yaml:3595]: microservice

File: src/Tooltip/TooltipDictionary.yaml [hunk 2]
```diff
@@ -3617,10 +3615,10 @@ selectFeature:
     content: Select the APM or logging tool feature to be used.
     width: "400"
 NameIdDescriptionTagsHealthSource_description:
-    content: It is a text string.
+    content: Enter a description.
     width: "400"
```
Comment (SmritiSatya) [inline src/Tooltip/TooltipDictionary.yaml:3619]: Can we say what description it is? Like "Enter a description for the ...".

File: src/Tooltip/TooltipDictionary.yaml [hunk 3]
```diff
@@ -3617,10 +3615,10 @@ selectFeature:
...
 NameIdDescriptionTagsHealthSource_tags:
-    content: You can add Tags to Harness entities and then use the Tags to search for all matching entities.
+    content: You can assign an existing tag or create a new tag. Tags are metadata and they can contain any character. You can use tags to search and filter the monitored service.    
     width: "400"
```
Comment (SmritiSatya) [inline src/Tooltip/TooltipDictionary.yaml:3622]: Assign an existing tag or create a new tag for the entities. Tags are metadata, and they can contain any character. You can use tags to search and filter the monitored services.

**Review by SmritiSatya — APPROVED** [review]

File: src/Tooltip/TooltipDictionary.yaml (hunk 3 above)
Comment (SudheendraKatte) [reply src/Tooltip/TooltipDictionary.yaml:3622]: Incorporated.

File: src/Tooltip/TooltipDictionary.yaml (hunk 1 above)
Comment (SudheendraKatte) [reply src/Tooltip/TooltipDictionary.yaml:3595]: Incorporated.

**Review by SmritiSatya — APPROVED** [review]

**Pull request merged**